cloud/aws/deploy/deploy.go

```diff
@@ -92,6 +100,7 @@ type NitricAwsPulumiProvider struct {
 	Secrets               map[string]*secretsmanager.Secret
 	Buckets               map[string]*s3.Bucket
 	BucketNotifications   map[string]*s3.BucketNotification
+	Distributions         map[string]*cloudfront.Distribution
```

Comment:
This should probably just be `*cloudfront.Distribution` rather than a map. I don't see us deploying more than one distribution and even if we do its easy enough to work in and we can properly consider how we'll uniquely identify each distribution at that time.

Reply:
Made some updates for this.